pyramid_swagger/tween.py

```diff
@@ -561,6 +561,11 @@ def get_op_for_request(request, route_info, spec):
         op = spec.get_op_for_request(request.method, route_path)
```

Comment:
I find this logic flow slightly counter-intuitive. Can we just move the lowest-level raise up into this if block?